test/ts/station_test.exs

```diff
@@ -844,7 +844,7 @@ defmodule StationTest do
 		# Send 10_000 queries
 		send_message(query, 10_000, pid)
 
-		:timer.sleep(1000)
+		:timer.sleep(5000)
```

Comment:
@suryapratapmehra bumping up this number to 10_000 should make the test pass on travis.